wp-content/themes/csisnuclearnetwork/assets/_scss/pages/archive.scss

```diff
@@ -103,129 +103,95 @@
 
   &__base {
     @include shadow('lg');
-
-    @include breakpoint('small') {
-      margin-top: rem(32);
-    }
-
-    @include breakpoint('medium') {
-      margin-top: rem(40);
-    }
-
-    @include breakpoint('large') {
-      margin-top: rem(56);
-    }
+    margin-top: rem(32);
   }
 }
 
-.pagination {
-  &,
-  &__results {
-    @extend %text-style-label-medium;
-    font-weight: normal;
-  }
-
-  &__results {
-    margin-bottom: rem(40);
-    padding-bottom: rem(8);
-    color: $color-black-190;
-    border-bottom: 1px solid $color-border-dark-130;
+.pagination__results {
+  @extend %text-style-label-medium;
+  padding-bottom: rem(16);
+  color: $color-black-180;
+  font-weight: normal;
+  border-bottom: 1px solid $color-border-dark-130;
+}
 
-    @include breakpoint('large') {
-      padding-bottom: rem(16);
-    }
-  }
+.pagination__wrapper {
+  display: flex;
+  gap: rem(16);
```

Comment:
I don't see this in the mockup and I think removing it will help a little with the overflow on the smallest screen size.

Reply:
I removed the other two breakpoints and changed this to rem(12). That way it gives the effect of the mock but does not break on mobile. This also looks like one bar rather than separate elements like before.